### PR TITLE
Exit HashBuild::addInput early if all join keys are null

### PR DESCRIPTION
**Pull request by mbasmanova**

### CODE REVIEW

File: velox/exec/HashBuild.cpp [hunk 1]
```diff
@@ -140,6 +140,11 @@ void HashBuild::addInput(RowVectorPtr input) {
     }
   }
 
+  if (!activeRows_.hasSelections()) {
```
Comment (xiaoxmeng) [inline velox/exec/HashBuild.cpp:143]: Do we want to add a unit test to cover this? Thanks!

File: velox/exec/HashBuild.cpp [hunk 2]
```diff
@@ -140,6 +140,11 @@ void HashBuild::addInput(RowVectorPtr input) {
...
+    // All join keys are null. No further processing is needed.
```
Comment (xiaoxmeng) [inline velox/exec/HashBuild.cpp:144]:
```
nit: VELOX_DCHECK(!isLeftJoin() || !isLeftSemiJoin);
```

**Review by xiaoxmeng — APPROVED** [review]
LGTM. Thanks for the change!

**Comment by mbasmanova** [comment]
Abandoning in favor of #2187

**Pull request closed without merging**